III.tdd-katas/fizz-buzz/fizz-buzz.js

```diff
@@ -0,0 +1,23 @@
+module.exports =function fizzbuzz(number){
+   
+
+    for (i = 1; i <= 100; i++) {
+        // if the number is multplie by 3 & 5, write "FizzBuzz"
+       
+        if (i % 15=== 0) {
+            return "FizzBuzz";
+        }
+        // if the number is multiple by 3, write "Fizz"
+        else if (i % 3 === 0) {
```

Comment:
Do we need all these `else` statements?
If we are already returning a value inside the `if` block, the function will finish its execution.